modules/custom/az_flexible_page/config/install/views.view.az_page_by_category.yml

```diff
@@ -186,19 +139,14 @@ display:
           relationship: none
           group_type: group
           admin_label: ''
+          entity_type: node
+          entity_field: title
+          plugin_id: standard
           order: ASC
-          exposed: false
           expose:
```

Comment:
double check: this is supposed to be 'expose', not 'exposed', right?

Reply:
do we need an `exposed: true`?

Reply:
This was exported directly from the site. So I can't directly comment on the `expose` vs `exposed` question.

And I believe having `expose: ` allows the filters to be displayed in a block rather than on the view page.